cf/domain.py

```diff
@@ -1,6 +1,9 @@
 from math import prod
 
 import cfdm
+import numpy as np
+
+import cf
```

Comment:
```suggestion
import cf
```
Why do we need to import this?